crates/sui-core/src/scoring_decision.rs

```diff
@@ -97,7 +97,10 @@ pub fn update_low_scoring_authorities(
         .for_each(|(a, s)| {
             let name = AuthorityName::from(a);
             debug!("authority {} has score {}", name, s);
-            metrics.consensus_handler_scores.observe(*s as f64);
+            metrics
+                .consensus_handler_scores
+                .with_label_values(&[&format!("{:?}", a)])
```

Comment:
any reason you would want to use `a`(BLS12381PublicKey)  instead of `name`(AuthorityPublicKeyBytes)

Reply:
is AuthorityPublicKeyBytes easier to read?